deployment/E_heroku.md

```diff
@@ -1 +1,308 @@
-## FPO
+### What we'll need
+
+The only thing we'll need for this guide is a working Phoenix application.
+
+If you don't already have one, just run `mix phoenix.new hello_phoenix` in a terminal and you're good to go.
+
+### Goals
+
+Our main goal for this is guide is to get a Phoenix application running on Heroku.
+
+## Steps
+
+Let's separate this process into a few steps so we can keep track of where we are.
+
+- Sign up for Heroku
+- Install the Heroku Toolbelt
+- Create the Heroku application
+- Add the Phoenix static buildpack
+- Make our project Heroku-ready
+- Deploy time!
+- Useful Heroku commands
+
+## Sign up for Heroku
+
+Signing up to Heroku is very simple, just head over to [https://signup.heroku.com/www-header](https://signup.heroku.com/www-header) and fill in the form.
+
+The Free plan will give us one web [dyno](https://devcenter.heroku.com/articles/dynos#dynos) and one worker dyno, as well as a PostgreSQL and Redis instance for free.
+
+These are meant to be used for testing and development, and come with some limitations. If you plan to run some serious business on Heroku you should definitely considering upgrading to a paid plan.
+
+## Install the Heroku Toolbelt
+
+Once you have signed up download the Heroku Toolbelt installer for your platform at [https://toolbelt.heroku.com/](https://toolbelt.heroku.com/).
```

Comment:
I think this needs a little smoothing out. How about something like this:

Once we have signed up, we can download the correct version of the Heroku Toolbelt for our system [here](https://toolbelt.heroku.com/).